config_test.go

```diff
@@ -1,57 +1,165 @@
 package main
 
 import (
-	"path"
+	"bytes"
+	"log"
 	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
 )
 
 func Test_Load(t *testing.T) {
-	tt := []struct {
-		Path string
-		Err  error
-	}{
-		{
-			"valid_config.json",
-			nil,
-		}, {
-			"valid_config_extras.json",
-			nil,
-		}, {
-			"invalid_config_author.json",
-			ErrAuthorRequired,
-		}, {
-			"invalid_config_email.json",
-			ErrEmailRequired,
-		},
-	}
-
-	for _, tc := range tt {
-		path := path.Join("testdata", tc.Path)
-		_, err := LoadConfigFile(path)
-		if err != tc.Err {
-			t.Fatalf("Load(%v) returned an error (%v)", path, err)
+	Convey("When loading a valid config file", t, func() {
+		fs = mockFS{
```

Comment:
ok good to know.  I think we're straying into overkill territory but let's discuss. 

semirelated: I'm looking through `func Test_ShowLog(t *testing.T)` ... line one declares `rc` to store `mockClock`but I do not see it being used elsewhere throughout the project.